src/main/java/net/mcreator/ui/modgui/CustomGUIGUI.java

```diff
@@ -36,6 +37,7 @@
 public class CustomGUIGUI extends ModElementGUI<GUI> {
 
 	private WYSIWYGEditor editor;
+	private GridSettings gridSettings;
```

Comment:
Move this to data model, not to GUI of mod element.

Reply:
What data model?

Reply:
GUI.java is data model of GUI mod element

Reply:
https://www.geeksforgeeks.org/mvc-design-pattern/#:~:text=The%20Model%20View%20Controller%20(MVC,but%20not%20for%20complete%20application. 

Data model here are mod element classes, view are ModElementGUI classes and generator system is controller. MCreator follows MVC model.